lumigator/python/mzai/backend/backend/api/router.py

```diff
@@ -12,3 +20,6 @@
 api_router.include_router(experiments.router, prefix="/experiments", tags=[Tags.EXPERIMENTS])
 api_router.include_router(completions.router, prefix="/completions", tags=[Tags.COMPLETIONS])
 api_router.include_router(models.router, prefix="/models", tags=[Tags.MODELS])
+api_router.include_router(
+    experiments_new.router, prefix="/experiments_new", tags=[Tags.EXPERIMENTS_NEW]
```

Comment:
This is where it could make sense to introduce API versioning, since we´re touching known endpoints, but I would leave it out of the scope of this PR and focus on the logic/job stitching first.